gcloud-java-contrib/gcloud-java-nio/src/test/java/com/google/gcloud/storage/contrib/nio/CloudStorageFileSystemProviderTest.java

```diff
@@ -547,115 +633,120 @@ public void testIsDirectory() throws IOException {
   }
 
   @Test
-  public void testIsDirectory_trailingSlash_alwaysTrue() {
-    assertThat(Files.isDirectory(Paths.get(URI.create("gs://military/fundir/")))).isTrue();
+  public void testIsDirectory_trailingSlash_alwaysTrue() throws IOException {
+    try (FileSystem fs = helper.forBucket("military")) {
+      assertThat(Files.isDirectory(fs.getPath("/fundir/"))).isTrue();
+    }
   }
 
   @Test
-  public void testIsDirectory_trailingSlash_pseudoDirectoriesDisabled_false() {
-    try (CloudStorageFileSystem fs = forBucket("doodle", usePseudoDirectories(false))) {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Method names should not contain underscores](https://www.codacy.com/app/mziccard/gcloud-java/file/2333823942/issues/source?bid=3123107&fileBranchId=3190440#l643)